### ISSUE TITLE
Use of a string in float field

### ISSUE THREAD

**Issue by damccull** [richardthombs/scunpacked#81]
In FlightCharacteristics, some ships have a String type where most others have a float. This breaks strongly-typed languages. While possible to parse, it makes things much more difficult. 

May I suggest producing only a single type per field across all ships/items/etc. It's fine to make them different across different entities, but a single entity should only have a single type. If the "Inifinite" value is needed, might there be another way to represent it besides changing the data type of the field?

**Comment by richardthombs** [comment]
Can you give an example? The JSON all comes from serialized .NET classes which are strongly typed, so there should not be any fields which are sometimes strings and sometimes numbers.

**Comment by damccull** [comment]
Hey, again I appreciate your revisiting these old issues. I'm honestly not 100% sure what exact examples I was talking about but I've learned a ton more about json since and I'm pretty sure what I was talking about is 64 bit integers getting turned into strings in json, which, from what I now understand, is desired behavior. It helps languages without native 64bit int support to be able to somehow parse them,I guess. I don't really understand why, to be honest, since it seems to me that json is all a string anyways and a language without 64bit int support could just simply parse a number that's too long into a string anyways. (looking at you, Java).

Thanks! I'm pretty sure I've worked out how to handle what's already there.
